src/unix/mod.rs

```diff
@@ -1067,7 +1067,11 @@ extern "C" {
     )]
     pub fn realpath(pathname: *const ::c_char, resolved: *mut ::c_char) -> *mut ::c_char;
 
-    pub fn flock(fd: ::c_int, operation: ::c_int) -> ::c_int;
+    cfg_if! {
```

Comment:
I think we usually use `#[cfg_attr]` rather than `cfg_if` where possible.  It looks like that in this file at least.

Reply:
Yeah, clippy flagged it on the first run so I changed to `cfg_if!`.  I've no strong feelings either way.